content/en/synthetics/platform/settings/_index.md

```diff
@@ -47,7 +47,23 @@ On the [Synthetic Monitoring & Continuous Testing Settings page][1], you can acc
 
 ### Enforced tags settings
 
-On the Usage Attribution page, you can configure up to three tags by which to break down cost and usage attributes. Select **Enforce tags for usage attribution on all tests** to require that users enter all configured Usage Attribution tags when creating or editing Synthetic tests. With this setting enabled, users cannot save tests without entering all required tags.
+Enforce tags for **usage attribution** on all tests:
+
+* On the Usage Attribution page, you can configure up to three tags by which to break down cost and usage attributes. Select **Enforce tags for usage attribution on all tests** to require that users enter all configured Usage Attribution tags when creating or editing Synthetic tests. With this setting enabled, users cannot save tests without entering all required tags. 
+
+Enforce required **monitor tag policies** on all tests:
+
+* On the [Synthetic Monitoring and Testings settings][20] page, select **Enforce required monitor tag policies on all tests** to require that users who set monitor tag policies will now have those tags enforced on Synthetic tests. With this setting enabled, users cannot save tests without entering all required tags.
+
+</br>
+
```

Comment:
```suggestion
```

Reply:
This seems to be preventing the numbered list below from being indented (which it looks like it's intended to be).

If that's **not** the intended effect, I think you can update your `</br>` tag to just `<br>` or `<br/>`, as it's a self-closing tag.

Reply:
thanks @buraizu ! Took a bit of fiddling but I think I got it now, can you re-review ?